src/main/python/fever/evaluate_doc_retrieval.py

```diff
@@ -59,7 +59,7 @@ def evaluate(query_id, predicted_docs):
         curr_query = -1
         predicted_docs = []
         for i, line in enumerate(f):
-            query_id, doc_id, rank = line.strip().split('\t')
+            query_id, _, doc_id, rank, _, _ = line.strip().split(' ')
```

Comment:
Can't we just go with `line.strip().split()`

Reply:
that should work too, i guess what's currently there is more explicit about what exactly we're splitting on

Reply:
Yeah, I see people arbitrarily picking between tabs/spaces in this case so that's why I was wondering if we are consistent here it is fine.